Modules/Axis/Sources/Axis/String/UTF8.swift

```diff
@@ -0,0 +1,54 @@
+extension UTF8 {
+    public static var whitespaceAndNewline: Set<UTF8.CodeUnit> = [32, 10, 9, 13]
```

Comment:
It would we nice if we could avoid this code duplication. Is this extension necessary, since we can just do `UnicodeScalars.whitespaceAndNewline.utf8()`?